domains/Discussions/Tests/Feature/AnswersIndexControllerTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Carbon\Carbon;
+use Illuminate\Http\Response;
+use Tests\TestCase;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Discussions\Database\Factories\AnswerFactory;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+
+class AnswersIndexControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private User $user;
+    private User $secondUser;
+    private Question $question;
+    private Answer $answer;
+    private Answer $secondAnswer;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user = UserFactory::new()->create();
+        $this->secondUser = UserFactory::new()->create();
+
+        $this->question = QuestionFactory::new([
+            'author_id' => $this->user->id
+        ])->create();
+
+        $this->answer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->user->id,
+            'created_at' => Carbon::now()->subWeek()->toDateTimeString()
+        ])->create();
+
+        $this->secondAnswer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->secondUser->id,
+            'created_at' => Carbon::now()->toDateTimeString()
+        ])->create();
+    }
+
+    /** @test */
+    public function it_gets_paginated_answers_for_a_question(): void
+    {
+        $this->get(route('discussions.questions.answersList', ['questionId' => $this->question->id]))
+            ->seeJsonStructure([
+                "data" => [
+                    0 => [
```

Comment:
Nit: we don't need to declare the array key.

Reply:
Actually I can't get the test to pass without those keys :S.
The $responseData that's getting to the seeJsonStructure function is empty, even though, if i put a dd on the method I want to test, answers are found.